### PR TITLE
Update DEVDOC.md for delve

### PR DESCRIPTION
**Pull request by lpy**
Add an example of delve command for remote debugging in VSCode.

Bug: N/A

### CODE REVIEW

**Comment by lpy** [comment]
I have also verified this on Mac

File: DEVDOC.md [hunk 1]
```diff
@@ -152,14 +152,17 @@ To use the delve debugger for Go with VSCode to debug `gapis`. These steps can b
...
-A sample `<host>` can be `127.0.0.1` and `<port>` can be `1234`.
+An example `<host>` can be `127.0.0.1` and `<port>` can be `1234`.
 
 4. Start delve in headless mode at gapid check-in folder.
 ```
 dlv exec --headless --listen=<host>:<port> --api-version 2 ./bazel-bin/pkg/gapis -- <gapis-arguments>
 ```
 
-This will allow using port `1234` (or any other preferred port) to connect to delve from VSCode.
+Below commnad will allow using port `1234` (or any other preferred port) to connect to delve from VSCode.
```
Comment (hevrard) [inline DEVDOC.md]: typo "commnad", and maybe rephrase: "The command below will allow using ..."

File: DEVDOC.md [hunk 2]
```diff
@@ -152,14 +152,17 @@ To use the delve debugger for Go with VSCode to debug `gapis`. These steps can b
...
+```
+dlv exec --headless --listen=127.0.0.1:1234 --api-version 2 ./bazel-bin/pkg/gapis -- -persist -rpc localhost:8888
```
Comment (yalcinmelihyasin) [inline DEVDOC.md]: This is redundant. See the line 159. Please remove it.

File: DEVDOC.md (hunk 1 above)
Comment (yalcinmelihyasin) [reply DEVDOC.md]: +1

File: DEVDOC.md [hunk 3]
```diff
@@ -152,14 +152,17 @@ To use the delve debugger for Go with VSCode to debug `gapis`. These steps can b
     ...
 }
 ```
-A sample `<host>` can be `127.0.0.1` and `<port>` can be `1234`.
+An example `<host>` can be `127.0.0.1` and `<port>` can be `1234`.
```
Comment (paulthomson) [inline DEVDOC.md]: Nit: As an example, `<host>` could be `127.0.0.1` and `<port>` could be `1234`.
